src/frontends/pytorch/src/utils.cpp

```diff
@@ -40,15 +41,23 @@ namespace pytorch {
 
 using namespace ov::op;
 
-void num_inputs_check(const NodeContext& context, size_t min_inputs, size_t max_inputs) {
+void num_inputs_check(const NodeContext& context, size_t min_inputs, size_t max_inputs, bool allow_complex) {
     auto num_inputs = context.get_input_size();
     FRONT_END_OP_CONVERSION_CHECK(num_inputs >= min_inputs,
                                   "Got less inputs ",
                                   num_inputs,
                                   " than expected ",
                                   min_inputs);
+    if (!allow_complex) {
+        // verify that no input is complex
+        for (size_t i = 0; i < std::min(num_inputs, max_inputs); i++) {
```

Comment:
```suggestion
        for (size_t i = 0; i < std::min(num_inputs, max_inputs); ++i) {
```
avoid copy of `i`

Reply:
also I advice to create `int required_inputs_num = static_cast<int>(std::min(num_inputs, max_inputs))`

this way you don't need casting inside the loop